JSONAPI/Json/JsonApiFormatter.cs

```diff
@@ -180,6 +180,13 @@ protected void Serialize(object value, Stream writeStream, JsonWriter writer, Js
         {
             writer.WriteStartObject();
 
+            //FIXME: The spec no longer requires that the ID key be "id":

```

Comment:
I think that any property that is being used as the id property should be serialized as "id" only.